src/sentry/grouping/component.py

```diff
@@ -41,8 +41,6 @@ def __init__(
         contributes: bool | None = None,
         values: Sequence[str | GroupingComponent] | None = None,
         variant_provider: bool = False,
-        is_prefix_frame: bool = False,
-        is_sentinel_frame: bool = False,
```

Comment:
The changes up until is point are obvious - they're just removing all the hierarchical stuff. What is this prefix/sentinel frame business about? Could you maybe explain in a sentence or two in the PR description why it's going?

Reply:
This all related to hierarchical grouping. I'm going to comment further in the main PR so we have it for posterity.